config/toml.go

```diff
@@ -343,12 +343,12 @@ dial_timeout = "{{ .P2P.DialTimeout }}"
 #
 #  Possible types:
 #  - "flood" : concurrent linked list mempool with flooding gossip protocol
-#  (default)
 #  - "nop"   : nop-mempool (short for no operation; the ABCI app is responsible
 #  for storing, disseminating and proposing txs). "create_empty_blocks=false" is
 #  not supported.
+#  - "priority" : concurrent linked list mempool with priority queue (default)
 #  - "cat"   : content addressable mempool
-type = "flood"
+type = "priority"
```

Comment:
[no change needed] for future reference it would've been clearer to isolate this change to a distinct PR because setting the default mempool to priority is different from adding support for the CAT mempool.

Reply:
i can pull it out if it makes it easier

Reply:
It's fine to keep in this PR for now. Resolving.